packages/mint-components/src/components/sqm-country-field/useCountryField.tsx

```diff
@@ -0,0 +1,33 @@
+import { useLocale, useQuery } from "@saasquatch/component-boilerplate";
+import { gql } from "graphql-request";
+
+export type Country = {
+  countryCode: string;
+  displayName: string;
+};
+
+const GET_COUNTRIES = gql`
+  query countryList($locale: RSLocale) {
+    countries(limit: 250) {
+      data {
+        countryCode
+        displayName(locale: $locale)
+      }
+    }
+  }
+`;
+
+export function useCountryField() {
+  const locale = useLocale();
```

Comment:
This isnt going to work as we dont really have a user yet, they aren't logged in